brave/brave-core
# Change Background Settings for New Tabs Page

Feature: https://github.com/brave/brave-browser/issues/4523
Sister PR to https://github.com/brave/brave-ui/pull/485 which was merged and reverted.
Sister PR to https://github.com/brave/brave-ui/pull/491 (brave-ui)

close https://github.com/brave/brave-browser/issues/4523

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- Ensure settings menu pop up properly (toggle)
- Ensure the background toggles from the menu
- Clicking outside the settings menu should close it

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
